// This lockfile was autogenerated by Pants. To regenerate, run:
//
//    pants generate-lockfiles --resolve=python-default
//
// --- BEGIN PANTS LOCKFILE METADATA: DO NOT EDIT OR REMOVE ---
// {
//   "version": 3,
//   "valid_for_interpreter_constraints": [
//     "CPython==3.12.*"
//   ],
//   "generated_with_requirements": [],
//   "manylinux": "manylinux2014",
//   "requirement_constraints": [
//     "numpy==2.1.1",
//     "pandas==2.2.3",
//     "python-dateutil==2.9.0.post0",
//     "pytz==2024.2",
//     "six==1.16.0",
//     "tzdata==2024.1"
//   ],
//   "only_binary": [],
//   "no_binary": []
// }
// --- END PANTS LOCKFILE METADATA ---

{
  "allow_builds": true,
  "allow_prereleases": false,
  "allow_wheels": true,
  "build_isolation": true,
  "constraints": [
    "numpy==2.1.1",
    "pandas==2.2.3",
    "python-dateutil==2.9.0.post0",
    "pytz==2024.2",
    "six==1.16.0",
    "tzdata==2024.1"
  ],
  "locked_resolves": [],
  "only_builds": [],
  "only_wheels": [],
  "path_mappings": {},
  "pex_version": "2.3.1",
  "pip_version": "24.0",
  "prefer_older_binary": false,
  "requirements": [],
  "requires_python": [
    "==3.12.*"
  ],
  "resolver_version": "pip-2020-resolver",
  "style": "universal",
  "target_systems": [
    "linux",
    "mac"
  ],
  "transitive": true,
  "use_pep517": null
}
